fzf-docker.plugin.zsh

```diff
@@ -28,10 +28,14 @@ _fzf_complete_docker() {
     _fzf_complete "--multi --header-lines=1" "$@" < <(
       docker images --format "table {{.ID}}\t{{.Repository}}\t{{.Tag}}\t{{.Size}}"
     )
-  elif [[ $ARGS == 'docker stop'* || $ARGS == 'docker rm'* || $ARGS == 'docker exec'* || $ARGS == 'docker kill'* || $ARGS == 'docker restart'* || $ARGS == 'docker logs'* ]]; then
+  elif [[ $ARGS == 'docker stop'* || $ARGS == 'docker exec'* || $ARGS == 'docker kill'* || $ARGS == 'docker restart'* || $ARGS == 'docker logs'* ]]; then
     _fzf_complete "--multi --header-lines=1 " "$@" < <(
       docker ps --format "${FZF_DOCKER_PS_FORMAT}"
     )
+  elif [[ $ARGS == 'docker rm'* ]]; then
+    _fzf_complete "--multi --header-lines=1 " "$@" < <(
+      docker ps -a | grep Exit --format "${FZF_DOCKER_PS_FORMAT}"
```

Comment:
Did you mean this? Or did I miss out something? :grin: 

```suggestion
      docker ps -a --format "${FZF_DOCKER_PS_FORMAT}" | grep Exit
```

Reply:
Oh, I think you meant `docker ps -a | grep Exit`

Reply:
What about `docker ps -a` without the grep? Why would we want only exited images?

Reply:
To be able to remove a container it already has to be stopped. You can still force it I think by adding -f. It's just my workflow to always stop it before removing it, so the running containers are not really relevant.
But you are right, it might be better to remove the `grep EXIT` to generalize it and leave the option.

Reply:
Sure, that would be a good option 😊 
Feel free to make the change, and I'll merge it right after 👍

Reply:
Done :)

Reply:
Thanks 😁